extension/src/config/klarna-constants.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  PAYMENT_METHOD_TYPE_KLARNA_METHODS: ['klarna', 'klarna_paynow', 'klarna_account'],
+  ADYEN_PERCENTAGE_MINOR_UNIT: 10000,
+  DEFAULT_PAYMENT_LANGUAGE: 'en',
```

Comment:
Why default is en ?

Reply:
Don't know why, but it's according to payone: https://github.com/commercetools/commercetools-payone-integration/blob/dfb14f4f17c356fa0d14b7f007aebe76065942ec/service/src/main/java/com/commercetools/util/ServiceConstants.java#L10

Reply:
Please clarify this, maybe it should not be default but mandatory.

Reply:
It's a language field that should be used for getting the name for a product. The shop can set the language on the payment or on the cart. If it's not set, it will pick a default language, which is 'en' in this case. If even 'en' is not there, it will just send "ITEM". I don't know why it's 'en' and not 'de' or 'fr' or some other language code, but I think 'en' is quite reasonable. It cannot be mandatory as the shop can choose any language they want. https://github.com/commercetools/commercetools-adyen-integration/pull/294/files#diff-1519468bcea5d71c0468f5405f237e61R24

Reply:
could you please add that decision as a comment, otherwise it's hard to reason.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/294/commits/419c4a50172ef8f2d667dc57bc5fd28eaadf0703#diff-4d2f466e11952575f6b8048a0f137b16R68-R72